code/modules/clothing/spacesuits/rig/modules/specific/ai_container.dm

```diff
@@ -54,14 +57,16 @@
 			SetupStat(rig)
 
 /obj/item/rig_module/ai_container/proc/update_verb_holder()
-	if(!verb_holder)
-		verb_holder = new(src)
+	if(!istype(verb_holder))
```

Comment:
`if(ispath(verb_holder))` would likely make more sense here since you're newing it immediately afterwards.